File: src/main/java/tw/racket/lang/psi/RacketTokenType.java
package tw.racket.lang.psi;

import com.intellij.lang.Language;
import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;
import tw.racket.lang.RacketLanguage;

public class RacketTokenType extends IElementType {
    public RacketTokenType(@NotNull String debugName, @Nullable Language language) {
        super(debugName, RacketLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "RacketTokenType." + super.toString();
    }
}

File: src/main/kotlin/tw/racket/lang/psi/RacketTokenType.kt
package tw.racket.lang.psi

import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import tw.racket.lang.RacketLanguage

class RacketTokenType(@NonNls debugName: String) : IElementType(debugName, RacketLanguage) {
    override fun toString(): String {
        return "RacketTokenType." + super.toString()
    }
}